### DISCUSSION TITLE
Compilation with cce

### DISCUSSION THREAD

**Discussion by csccva** [OpenMathLib/OpenBLAS#4226] [General]
Hello,

Has anyone managed to compile OpenBLAS with the cray compilers?

Cristian

**Comment by martin-frbg** [comment]
Support for (building with) them was added in 0.3.21, but I do not currently have access to an installation for testing. If you have tried and ran into problems, please open an issue ticket with the relevant log file(s).

**Comment by csccva** [comment]
Thank you for your reply. I tried both with spack and directly from source. I will open issue.

Cristian
